docs/specs/bot-protection.md

```diff
@@ -210,16 +210,13 @@ The bot protection behavior in generated flows depend on
 
 Given `bot_protection.enabled=true`,
 
-- `bot_protection.authentication_flow.signup_or_login.mode`
-  - Branches `identification: email`, `identification: phone`, `identification: username` of step `type: identify` in flows `signup`, `login`, `signup_login`, `promote` has `bot_protection.mode` equal to `bot_protection.authentication_flow.signup_or_login.mode`.
-- `bot_protection.authentication_flow.account_recovery.mode`
-  - All branches of `type: identify` of flow `account_recovery` has `bot_protection.mode` equal to `bot_protection.authentication_flow.account_recovery.mode`.
-- `bot_protection.authenticator.password.mode`
-  - Branches `authentication: primary_password`, `authentication: secondary_password` in step `type: authenticate` in flows `login` has `bot_protection.mode` equal to `bot_protection.authenticator.password.mode`.
-- `bot_protection.authenticator.oob_otp_email.mode`
-  - Branches `authentication: primary_oob_otp_email`, `authentication: secondary_oob_otp_email` in steps `type: authenticate`, `type: create_authenticator` in flows `signup`, `login`, `promote` has `bot_protection.mode` equal to `bot_protection.authenticator.oob_otp_email.mode`.
-- `bot_protection.authenticator.oob_otp_sms.mode`
-  - Branches `authentication: primary_oob_otp_sms`, `authentication: secondary_oob_otp_sms` in steps `type: authenticate`, `type: create_authenticator` in flows `signup`, `login`, `promote` has `bot_protection.mode` equal to `bot_protection.authenticator.oob_otp_sms.mode`.
+|Configuration|Behavior|
+|---|---|
+|`bot_protection.authentication_flow.signup_or_login.mode`|Branches `identification: email`, `identification: phone`, `identification: username` of step `type: identify` in flows `signup`, `login`, `signup_login`, `promote` has `bot_protection.mode` equal to `bot_protection.authentication_flow.signup_or_login.mode`.|
+|`bot_protection.authentication_flow.account_recovery.mode`|All branches of `type: identify` of flow `account_recovery` has `bot_protection.mode` equal to `bot_protection.authentication_flow.account_recovery.mode`.|
+|`bot_protection.authenticator.password.mode`|Branches `authentication: primary_password`, `authentication: secondary_password` in step `type: authenticate` in flows `login` has `bot_protection.mode` equal to `bot_protection.authenticator.password.mode`.|
+|`bot_protection.authenticator.oob_otp_email.mode`|Branches `authentication: primary_oob_otp_email`, `authentication: secondary_oob_otp_email` in steps `type: authenticate`, `type: create_authenticator` in flows `signup`, `login`, `promote` has `bot_protection.mode` equal to `bot_protection.authenticator.oob_otp_email.mode`.|
```

Comment:
Seems 

- flows `signup`, `promote`
     - branches `identification: email` `identification: phone` 
         - `type: identify` 

should equal to `bot_protection.authenticator.oob_otp_email.mode`

## Rationale

Currently AuthUI will send sms/email immediately after identify with `phone/email`.

### Screenshot Video

https://github.com/user-attachments/assets/466455f9-1386-4960-888b-180d541d58ce

Reply:
Added commit "Apply bot protection to signup verification as well" to clarify this case.